### DISCUSSION TITLE
Couchbase Lite for Dart support for named collections and scopes?

### DISCUSSION THREAD

**Discussion by jhilgedick-disney** [cofu-app/cbl-dart#527] [General]
Hello all!  Love Couchbase Lite for Dart.  Been playing with it for a few months now.  It seems to me that the current version of CBL for Dart does NOT support named collections & scopes (other than the default) and I was wondering if this was on the road map?  Or am I just wrong about this?

Thanks!

**Comment by blaugold** [comment]
Hey @jhilgedick-disney, you are right, that the current stable version does not yet support collections and scopes. 

Support for scopes and collections is already implemented and available as a [prerelease](https://pub.dev/packages/cbl/versions/3.0.0-dev.1). `3.0.0` will include a few more breaking changes (they are small though and should not affect most users) and the documentation needs a bigger update because of scopes and collections and I haven't had time yet to work on that. But it's very unlikely that the scopes and collections API will change in the stable release.
